### PR TITLE
introduce django-environ

### PR DESCRIPTION
**Pull request by artsyjian**
## Pull Request

**Description:**

- Introduce django-environ
- Switch to django-environ for most env vars that are read in settings.py.

**Related Issues:**
- https://github.com/SpokaneTech/SpokaneTech_Py/issues/21

**Checklist:**
- [ ] All tests pass.
- [ ] Code follows the project's coding standards.
- [ ] Documentation has been updated.

### CODE REVIEW

**Review by joeriddles — APPROVED** [review]
Looks great! Congrats on the first commit to Spokane Tech! Thank you :)

File: pyproject.toml [hunk 1]
```diff
@@ -18,6 +18,7 @@ dependencies = [
     "django-celery-beat>=2.6.0",
     "django-celery-results>=2.5.1",
     "django-crispy-forms>=2.1",
+    "django-environ>=0.11.2",
```
Comment (KGB33) [inline pyproject.toml:21]: When a new dependency is added the lock-files need to be regenerated. I wrote a snippet in the docs here: https://docs.spokanetech.org/CONTRIBUTING/#adding-dependencies

(Which, tbf, I wrote after you started.)

**Review by KGB33 — CHANGES_REQUESTED** [review]
Looks like CI is failing (I also just approve the Actions runs here, @me in the Discord or on here if they need approval again). 


```
$ dagger -m github.com/artsyjian/SpokaneTech_Py call linters all

PASSED! Command ruff check passed!
PASSED! Command bandit -c pyproject.toml -r src passed!

FAILED! Command `src/manage.py check` failed with exit code 1.
FAILED! Command `ruff format --check` failed with exit code 1.
FAILED! Command `pytest -vv --config-file pyproject.toml src` failed with exit code 1.
```

It looks like Pytest & Django **should** pass once the lockfiles are updated. 

Thanks for the contribution and let me know if you have any questions!

File: pyproject.toml (hunk 1 above)
Comment (artsyjian) [reply pyproject.toml:21]: Nice. Updated lock files.

**Review by KGB33 — APPROVED** [review]
LGTM.

**Pull request closed without merging**